kong/plugins/http-log/sender.lua

```diff
@@ -58,7 +58,12 @@ local function send(self, bodies)
         return false
       end
     end
-
+    
+    ok, err = httpc:set_keepalive(self.keepalive)
```

Comment:
`set_keepalive` must be called _after_ `request`, or else the connection will be considered closed from the Lua-land socket object by the time the request is made. See the documentation:

> This method also makes the current cosocket object enter the "closed" state, so there is no need to manually call the close method on it afterwards.

Reply:
I just keep taking L's lol. But at least I did recognize the nil pointer 😸 . Did another commit, should take care of everything now. If yall have a cleaner preference let me know.